### PR TITLE
Dont ignore import warnings

### PR DESCRIPTION
**Pull request by nikku**

### CODE REVIEW

File: test/integration/tck.js [hunk 1]
```diff
@@ -46,11 +46,17 @@ describe('dmn-moddle - TCK roundtrip', function() {
 
   const fileNames = glob(tckDirectory + '/TestCases/**/*.dmn', { cwd: __dirname });
 
+  function shouldRun(fileName) {
+    const match = process.env.GREP;
```
Comment (barmac) [inline test/integration/tck.js]: Clever!

**Comment by barmac** [comment]
> ```json
>     {
>       "name": "Size",
>       "superClass": [
>         "dc:Dimension"
>       ]
>     },
> ```

Good idea. I've pushed that on top.

**Comment by barmac** [comment]
Merged commits except for:
* https://github.com/bpmn-io/dmn-moddle/pull/11/commits/ab09ccd52a4e7bd19d6258d8bdc4c920bd6377d6
* https://github.com/bpmn-io/dmn-moddle/pull/11/commits/aa5aff7d568772468ece87c493e69a839588c0d4

**Comment by nikku** [comment]
Can you update this branch based on the latest release?

**Comment by barmac** [comment]
👍

**Comment by barmac** [comment]
Done :)

File: tasks/transforms/transformDMN.js [hunk 2]
```diff
@@ -51,6 +51,9 @@ module.exports = async function(results) {
 
   findProperty('ExtensionElements#extensionElement', model).name = 'values';
 
+  // redefine `Expression#binding`
```
Comment (nikku) [inline tasks/transforms/transformDMN.js]: For each of these fixes, can we add an actual integration test (diagram that works now and can be roundtripped)?

Comment (nikku) [reply tasks/transforms/transformDMN.js]: Ping.

Comment (barmac) [reply tasks/transforms/transformDMN.js]: Agreed.

Comment (philippfromme) [reply tasks/transforms/transformDMN.js]: Yes.

Comment (philippfromme) [reply tasks/transforms/transformDMN.js]: I'm having difficulties writing a test for this particular fix as it fixes only one of several warnings when parsing the diagrams.

**Comment by nikku** [comment]
Closing this in favor of https://github.com/bpmn-io/dmn-moddle/pull/12.

**Pull request closed without merging**